core/quantity/src/test/Parsing.test.ts

```diff
@@ -215,7 +210,7 @@ describe("Parsing tests:", () => {
       { value: "-2FT 6IN", quantity: { magnitude: -2.5, unitName: "Units.FT" } },
       { value: "1 1/2 FT", quantity: { magnitude: 1.5, unitName: "Units.FT" } },
       { value: "2FT 6IN", quantity: { magnitude: 2.5, unitName: "Units.FT" } },
-      { value: `2'-6"`, quantity: { magnitude: 2.5, unitName: "Units.FT" } },
+      { value: `2' 6"`, quantity: { magnitude: 2.5, unitName: "Units.FT" } },
```

Comment:
why did you have to change this case?

Reply:
add test case for same exact string where `-` is a `spacer`

Reply:
Done, thx